README.md

```diff
@@ -11,3 +11,154 @@ This Repository
 ===============
 
 This repository contains the markup needed to deploy the floeproject.org website.
+
+
+News Item Template
```

Comment:
I wonder if we need to add some more detail. Perhaps:

"... Save your renamed file to the news folder.

Once you have created the new file in the `./news/` directory, make sure to update:
* The `./news/index.html` page with a copy of the news post.
* The front page `./index.html` News section with a link and short description. There should be at most 3 news items on the front page. The newest news article appears first, and the oldest item is removed.